subsidy/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path('list', views.subsidy_list, name="subsidy_list"),
     path('create', views.subsidy_create, name="subsidy_create"),
-    path('<int:subsidy_id>/delete/', views.subsidy_delete, name='subsidy_delete'),
+    path('<int:subsidy_id>/update', views.subsidy_update, name='subsidy_update'),
```

Comment:
- I don't know if "Editar" button on list must redirect to edit page, but it doesn't work. I suppose that Frontend will carry out this point.

Reply:
We have decided to disconnect back to front, and at the end of sprint we will connect it